main/main.cpp

```diff
@@ -1810,6 +1812,9 @@ Error Main::setup2(Thread::ID p_main_tid_override) {
 	audio_server = memnew(AudioServer);
 	audio_server->init();
 
+	// init the Video decoder server
+	video_decoder_server = memnew(VideoDecoderServer);
```

Comment:
Does it actually need to be this early? For XR it's a special case as OpenXR needs to be set up before Vulkan, but I don't think that matters for video decoding?

Reply:
It used to be important when I had to set it up before the plugins loaded. It doesn't matter anymore. Where should I put it?

Actually I can just turn the whole class static since pretty much everything that matters is.